### PR TITLE
Small Deff update

### PR DESCRIPTION
**Pull request by Duny-**
- Re-added the security checkpoint in medbay. Closes https://github.com/d3athrow/vgstation13/issues/8639. 

ss 2016-03-14 at 06 33 21
- Added a few additional air alarms in hallways. Fixes https://github.com/d3athrow/vgstation13/issues/8757.
- Rotated the AME room's layout to allow people to build and expand the AME horizontally rather than vertically, as AME code is apparently shit and will fuck up if built vertically. Removes Deff tag on https://github.com/d3athrow/vgstation13/issues/8698.

ss 2016-03-14 at 06 34 41
- Added windoors behind plastic flaps in places where simply deconstructing plastic flaps would allow anyone to enter. Fixes https://github.com/d3athrow/vgstation13/issues/8611.
- Added a bar of soap in both custodial closets. Closes https://github.com/d3athrow/vgstation13/issues/8547.

### CODE REVIEW

**Comment by Imagebard** [comment]
>Security Checkpoint protected by plasma glass

Not necessary at all.

**Comment by Duny-** [comment]
\>people complain it's too easy to break in
\>simply change the glass to plasma glass
\>nobody complains
\>someone is drunk and removes the checkpoint
\>re-add it
\>complaints about plasmaglass
fucking normies

Why are my checks failing even though this shit has no conflicts and compiles fine for me reeeeee

File: html/changelogs/Duny.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
 author: Duny
-changes: []
+changes: 
+  - tweak: Deff: Re-added the security checkpoint in medbay.
```
Comment (9600bauds) [inline html/changelogs/Duny.yml]: You need to put these lines between "" if they have weird characters like : because it doesn't like them

**Comment by Imagebard** [comment]
>people complain it's too easy to break in

You can still break in with just a screwdriver and a crowbar. No idea what the plasma glass is here for.

**Comment by Duny-** [comment]
To annoy you by being purple

**Comment by 9600bauds** [comment]
Not everyone has a screwdriver on them I guess
Really though, what the fuck is the problem with the added security? It being purple?

**Comment by Imagebard** [comment]
Nothing. It just makes too much sense actually.
Medbay is blue, security is red. The border between them is purple.

*Red + Blue.*

**Pull request merged**